bundle/regal/rules/imports/unresolved-reference/unresolved_reference.rego

```diff
@@ -0,0 +1,122 @@
+# METADATA
+# description: Reference to unresolved field.
```

Comment:
"field" isn't commonly used in Rego terminology. I'd just say "Unresolved reference" I think, but open to suggestions!

Reply:
Also, skip the trailing period.